addons/sale/tests/test_sale_order.py

```diff
@@ -505,6 +505,26 @@ def test_sent_quotation_followers(self):
 
         self.assertIn(self.partner2, sale_order.message_partner_ids)
 
+    def test_scheduled_mark_so_as_sent(self):
+        """Check that a order gets marked as sent after a scheduled message was sent."""
+        self.env['mail.scheduled.message'].search([]).unlink()
```

Comment:
Why don't you use the `model` and `res_id` fields in the search below instead of unlinking everything here?

Reply:
Why do you need that at all anyway ?